Dockerfile

```diff
@@ -3,11 +3,9 @@ FROM debian:stable
 RUN apt-get update && \
```

Comment:
use `maven:3.9.1-eclipse-temurin-11-focal` and install perl and gnupg there.

Reply:
done